grpc/server.go

```diff
@@ -59,8 +73,19 @@ func (sb *serverBuilder) Add(desc *grpc.ServiceDesc, impl any) *serverBuilder {
 		sb.err = fmt.Errorf("attempted double-registration of gRPC service %q", desc.ServiceName)
 		return sb
 	}
+	sb.services[desc.ServiceName] = service{desc: desc, impl: impl}
+	return sb
+}
 
-	sb.services[desc.ServiceName] = service{desc, impl}
+// AddWithCheckInterval is the same as Add() but also sets the interval
+// used for long-running health checks.
+func (sb *serverBuilder) AddWithCheckInterval(desc *grpc.ServiceDesc, impl any, interval time.Duration) *serverBuilder {
```

Comment:
> In fact, this is sorta how the it already is, since you put the HealthCheckInterval as a top-level config item, rather than inside the GRPCServerConfig

🤦🏻‍♀️ oof, gotta admit I failed to notice I did this, good catch 👍🏻